### ISSUE TITLE
vip-manager complains virtual ip address's state is false

### ISSUE THREAD

**Issue by duj4** [cybertec-postgresql/vip-manager#83]
Hi Team,

I am using consul as my DCS and the version of vip-manager is 1.0.2. Once I tried to bring it up via "vip-manager -config=./vip-manager.yml", it posted:
```
IP address <ip>/<netmask> state is false, desired false
IP address <ip>/<netmask> state is false, desired true
Problems with producing the arp client: listen package <part of mac I think>: socket: operation not permitted
Problems with producing the arp client: listen package <part of mac I think>: socket: operation not permitted
too many retries
Couldn't create an Arp client: listen package <part of mac I think>: socket: operation not permitted

```
Does this mean I do not have the permission to add the VIP to the ens?

**Comment by duj4** [comment]
I tried to run vip-manager with root, all is good. It seems the previous user is lack of permission to do so?

**Comment by markwort** [comment]
Yes, you are absolutely right, superuser privileges are required if you want vip-manager to be able to register and deregister virtual IP addresses locally.

We should add this point to the documentation.

**Comment by markwort** [comment]
Please find the update to the docs here:
https://github.com/cybertec-postgresql/vip-manager#environment-prerequisites

Hope vip-manager is working well for you now!
